python/doc/pyplots/TensorProductExperiment.py

```diff
@@ -0,0 +1,24 @@
+import openturns as ot
+from matplotlib import pyplot as plt
+from openturns.viewer import View
+ot.RandomGenerator.SetSeed(0)
+
+collection = ot.WeightedExperimentCollection(2)
```

Comment:
here too

Reply:
Oups, I forgot that one. This is the last. Fixed.